pypesto/objective/amici_objective.py

```diff
@@ -569,6 +525,44 @@ def map_par_opt_to_par_sim(mapping_par_opt_to_par_sim, par_opt_ids, x):
     return par_sim_vals
 
 
+def create_plistfrom_par_opt_to_par_sim(mapping_par_opt_to_par_sim,
+                                        par_opt_ids):
+    """
+    From the optimization vector `x`, create the simulation vector according
+    to the mapping `mapping`.
+
+    Parameters
+    ----------
+
+    mapping_par_opt_to_par_sim: array-like of str
+        len == n_par_sim, the entries are either numeric, or
+        optimization parameter ids.
+    par_opt_ids: array-like of str
+        The optimization parameter ids. This vector is needed to know the
+        order of the entries in x.
+
+    Returns
+    -------
+
+    plist: array-like of float
+        List of parameter indices for which sensitivity needs to be computed
+    """
+    plist = []
+
+    # number of simulation parameters
+    n_par_sim = len(mapping_par_opt_to_par_sim)
+
+    # iterate over simulation parameter indices
+    for j_par_sim in range(n_par_sim):
```

Comment:
```suggestion
    for j_par_sim, val in enumerate(mapping_par_opt_to_par_sim):
```
... and remove what's before (playing codacy :) )